### ISSUE TITLE
V2 not in PyPI

### ISSUE THREAD

**Issue by IQNeoXen** [benediktschmitt/py-ts3#61]
First of all: Great API! Thank you very much!

It just seems that V2 is not in the PyPI. Doesn't matter what I do, I only get V1.0.6

`C:\WINDOWS\system32>pip3 install ts3 --upgrade --force-reinstall
Collecting ts3
  Using cached ts3-1.0.6.tar.gz
Collecting blinker (from ts3)
  Using cached blinker-1.4.tar.gz
Installing collected packages: blinker, ts3
  Found existing installation: blinker 1.4
    Uninstalling blinker-1.4:
      Successfully uninstalled blinker-1.4
  Running setup.py install for blinker ... done
  Found existing installation: ts3 1.0.6
    Uninstalling ts3-1.0.6:
      Successfully uninstalled ts3-1.0.6
  Running setup.py install for ts3 ... done
Successfully installed blinker-1.4 ts3-1.0.6`

**Comment by matobodo** [comment]
same issue here

**Comment by benediktschmitt** [comment]
You're right, I haven't uploaded v2 to PyPi yet, because I'm thinking about adding some new features. Nevertheless, I'll try to upload the current dev version to PyPi in the next days while keeping the v1 as default.

-- Sorry for the delay, I've been on holiday.
